### PR TITLE
OCPRHV-274 Add diagram of OCP cluster on RHV cluster

### PR DESCRIPTION
**Pull request by rolfedh**

### CODE REVIEW

File: installing/installing_rhv/installing-rhv-customizations.adoc [hunk 1]
```diff
@@ -5,7 +5,11 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-You can customize and install a cluster on {rh-virtualization-first}. The installation program uses installer-provisioned infrastructure to automate creating and deploying the cluster.
+You can customize and install a {product-title} cluster on {rh-virtualization-first}, similar to the one shown here.
```
Comment (jboxman) [inline installing/installing_rhv/installing-rhv-customizations.adoc:8]: s/here./in the following diagram./

**Review by jboxman — COMMENTED** [review]
One thought.

**Comment by jboxman** [comment]
@rolfedh, the title of the PR is interesting; Is there a particular convention in play here?

**Review by peterclauterbach — APPROVED** [review]
LGTM

**Comment by rolfedh** [comment]
@jboxman Good to merge and cherry-pick.

**Pull request merged**